aws/variables.tf

```diff
@@ -12,15 +12,15 @@ variable "humio_instance_count" {
 }
 variable "kafka_instance_type" {
   type    = string
-  default = "kafka.m5.large"
```

Comment:
I think we want to keep this as m5.large. The t3s run out of credits very fast if there is any kind of load thrown at them.

Reply:
I should have changed that in a local variables so I put it back. It worth pointing out in docs that default sizes are probably not right for anyone instance sizes and counts need to be calculated based on needs